src/features/vote/proposals/list/ProposalInterestVote.tsx

```diff
@@ -0,0 +1,95 @@
+import { Label } from '@radix-ui/react-label';
+import { MinusIcon, ThumbsDownIcon, ThumbsUpIcon } from 'lucide-react';
+import {
+  RadioGroup,
+  RadioGroupItem,
+} from '../../../../components/ui/radio-group.tsx';
+import { cn } from '../../../../utils.ts';
+import type { Proposal, Vote } from '../types.ts';
+
+const voteOptions = [
+  {
+    label: 'Strongly Disinterested',
+    value: '-2',
+    id: 'vote-strongly-disinterested',
+  },
+  {
+    label: 'Slightly Disinterested',
+    value: '-1',
+    id: 'vote-slightly-disinterested',
+  },
+  { label: 'Neutral', value: '0', id: 'vote-neutral' },
+  { label: 'Slightly Interested', value: '1', id: 'vote-slightly-interested' },
+  { label: 'Strongly Interested', value: '2', id: 'vote-strongly-interested' },
+];
```

Comment:
POLISH: I think we can shorten some of the labels on these
SUGGESTION: 
```suggestion
const voteOptions = [
  {
    label: 'Strong Disinterest',
    value: '-2',
    id: 'vote-strong-disinterest',
  },
  {
    label: 'Slight Disinterest',
    value: '-1',
    id: 'vote-slight-disinterest',
  },
  { label: 'Neutral', value: '0', id: 'vote-neutral' },
  { label: 'Slight Interest', value: '1', id: 'vote-slight-interest' },
  { label: 'Strong Interest', value: '2', id: 'vote-strong-interest' },
];
```

Reply:
I like this suggestion! I've updated based on input.